tests/unit/fem/test_conv.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
Great, thanks! Do we want to add this to test_pa_kernels.cpp? The name of that test seems generic enough :)

Reply:
Sorry, test_pa_kernels.cpp is not in `master` yet. I'm referring to the test in `vecmass-vecdiff-dev`.

Reply:
Sure we can add it into test_pa_kernels.cpp, https://github.com/mfem/mfem/pull/1167  will be merged into master soon right? I can shuffle the unit test around when it is in.